nutrition_labels/grant_tagger.py

```diff
@@ -76,18 +87,34 @@ def evaluate(self, X, y):
         print(classification_report(y, y_predict))
         print(confusion_matrix(y, y_predict))
 
+    def return_mislabeled_data(self,y_test,y_pred):
+        X_test = [self.X[i] for i in self.test_indices]
```

Comment:
I would make this generic to either the train or test data by adding another argument, i.e. 
```
def return_mislabeled_data(self, y_actual, y_pred, X_indices):
       X_text = [self.X[i] for i in X_indices]
       X_text_df = pd.DataFrame({'Description': X_text,
                                'True_label':y_actual,
                                'Predicted_label':y_pred})
```

Reply:
this is a much better way of doing it thanks :)